File: app/src/main/java/com/BlizzardArmory/diablo/Item/RandomAffix.java

package com.BlizzardArmory.diablo.Item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;


public class RandomAffix {

    @SerializedName("oneOf")
    @Expose
    private List<OneOf> oneOf;

    public List<OneOf> getOneOf() {
        return oneOf;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/diablo3/item/RandomAffix.kt
package com.BlizzardArmory.model.diablo.diablo3.item

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Random affix.
 */
@Keep
data class RandomAffix(

    @SerializedName("oneOf")

    val oneOf: List<OneOf>
)